server/src/main/resources/db/dev/R__Load_testing_data.sql

```diff
@@ -34,6 +34,16 @@ INSERT INTO member_profile -- Gina Bremehr
 VALUES
 ('01b7d769-9fa2-43ff-95c7-f3b950a27bf9', PGP_SYM_ENCRYPT('Gina','${aeskey}'), PGP_SYM_ENCRYPT('Bremehr','${aeskey}'), PGP_SYM_ENCRYPT('COO','${aeskey}'), PGP_SYM_ENCRYPT('St. Louis','${aeskey}'), PGP_SYM_ENCRYPT('bremehrg@objectcomputing.com','${aeskey}'), '12312345', '2012-09-20', PGP_SYM_ENCRYPT('Epitome of Strong Woman','${aeskey}'), null, '1988-09-21');
 
+INSERT INTO member_profile -- Big Boss
```

Comment:
I changed some `member_profile` records so they have `supervisorid` values. This was needed to test some if the changes for this story.